swagger/definitions/request/public/message/update_payload.yml

```diff
@@ -2,19 +2,5 @@
 type: object
 properties:
   submitted_values:
-    oneOf:
```

Comment:
oneOf is not supported in 2.0. We will try to migrate this to 3.0 in the coming weeks.